modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,287 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
```

Comment:
It's not really "Creating an additional OVN-Kubernetes br-ex bridge network". It's more like:
Configuring OVN-Kubernetes to use a secondary OVS bridge

Reply:
:heavy_check_mark: